src/Microsoft.AspNetCore.Http.Connections.Client/Internal/WebSocketsTransport.Log.cs

```diff
@@ -69,6 +70,12 @@ private static class Log
             private static readonly Action<ILogger, Exception> _startedTransport =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(19, "StartedTransport"), "Started transport.");
 
+            private static readonly Action<ILogger, WebSocketMessageType, int, bool, string, Exception> _messageReceivedString =
+                LoggerMessage.Define<WebSocketMessageType, int, bool, string>(LogLevel.Debug, new EventId(20, "MessageReceivedString"), "Message received. Type: {MessageType}, size: {Count}, EndOfMessage: {EndOfMessage}. String data: {Message}.");
```

Comment:
Couldn't this be one event with the "type" (text or binary) as just another parameter?

Reply:
Yeah, I think when I first wrote it I was thinking I would print a byte[], but now they are both strings